Parser/asdl_c.py

```diff
@@ -687,19 +690,50 @@ def visitModule(self, mod):
         }
         res = PyObject_SetAttr(self, name, PyTuple_GET_ITEM(args, i));
         Py_DECREF(name);
-        if (res < 0)
+        if (res < 0) {
             goto cleanup;
+        }
     }
     if (kw) {
         i = 0;  /* needed by PyDict_Next */
+        if (_PyObject_LookupAttrId(fields, &PyId_index, &index) < 0) {
+            goto cleanup;
+        }
         while (PyDict_Next(kw, &i, &key, &value)) {
+            PyObject* pos = _PyObject_FastCallDict(index, &key, 1, NULL);
+            PyObject* err = PyErr_Occurred();
+            if (pos == NULL || err != NULL) {
```

Comment:
Just check for `pos == NULL`. An exception will be set if `_PyObject_FastCallDict()` returned `NULL`.

Reply:
Hi @remilapeyre Do you have time for a fix in your PR?
Thank you